scripts/fetch_tools.sh

```diff
@@ -233,6 +233,77 @@ gettool_tar-diff() {
     done
 }
 
+gettool_ginkgo() {
+    SCRIPT_DIR="$(dirname "${BASH_SOURCE[0]}")"
```

Comment:
We already have this defined in the top of the script.

Reply:
Thanks, removed.